docs/source/installation/linux.rst

```diff
@@ -17,6 +17,17 @@ Install manim via pypi::
     # pip3 install manimlib
 
 OR Install manim via the git repository with venv::
+NOTE : Make sure that the python version you are using is python3.7 because manim works only for that version . If not, you will get errors when running the old_projects.If the python3 version is not 3.7 then upgrade your version usimg the following steps:
```

Comment:
You'll actually get errors running things in old_projects no matter what.

Reply:
@eulertour Would you be able to specify the types of errors that one would normally be getting from the old_projects? (Disclaimer: Today marks my first day working with the repository and I'm on Debian.) I'm wondering if it's even worth debugging these with [https://documen.tician.de/pudb/](url).)

Reply:
Not really; they were written for an older and buggier version of manim, and some are even written for python 2 rather than 3. There's bound to be a lot of constructs used that don't exist in the current version.

Reply:
@ eulertour by old projects I mean the already exsisting projects like example_scenes.py . I would have specified it before